src/main/preprocessing/code_anonimization.py

```diff
@@ -27,7 +28,10 @@ def anonymize_code_in_df(df: pd.DataFrame) -> pd.DataFrame:
     # Todo: add other languages???
     # Delete incorrect fragments
     df = df[df.apply(lambda row: not is_incorrect_fragment(row[TESTS_RESULTS]), axis=1)]
-    df[TESTS_RESULTS] = df[TESTS_RESULTS].apply(lambda x: max(unpack_tests_results(x, TASK.tasks())))
+    file_name = df[FILE_NAME].unique()[0]
+    current_task = TASK(get_name_from_path(file_name, False))
```

Comment:
We do have a column with Task, please, use it here instead of file name

Reply:
In the new data, we don't have it